e2e/test/scenarios/admin/performance/dashboardsAndQuestions.cy.spec.ts

```diff
@@ -0,0 +1,321 @@
+            inheritsStrategyFrom: sampleDatabase,
+          },
+          {
+            description:
+              "dashboards can inherit duration strategy from instance-wide default policy",
+            strategy: sampleDurationStrategy,
+            item: sampleDashboard,
+            inheritsStrategyFrom: instanceDefault,
+          },
+        ] as CacheTestParameters[]
+      ).forEach(
+        ({
+          strategy,
+          item,
+          inheritsStrategyFrom,
+          description,
+          it = itFunction,
+        }) => {
+          it(description, () => {
+            const { reload, visitItem } =
+              item.model === "question"
+                ? setupQuestionTest(sampleQuestion)
+                : setupDashboardTest(sampleDashboard, sampleQuestion);
+
+            const itemToConfigure = inheritsStrategyFrom ?? item;
+
+            cy.then(function () {
+              selectCacheStrategy({
+                item: itemToConfigure,
+                strategy,
+              });
+
+              visitItem();
+            });
+
+            cy.then(function () {
+              reloadUntil(resultIsCached, {
+                logMessage: "Reloading until the result is cached",
+                wrapFinalResultAs: "cachedResult",
+                wrapTimeOfFinalResultAs: "firstCachedResultSeenAt",
+                reload,
+              });
+            });
+
+            const expectedCacheDuration =
+              getMillisecondsUntilCacheShouldBeInvalidated(strategy);
+            cy.then(function () {
+              getCacheDuration({
+                cacheStartTime: this.firstCachedResultSeenAt,
+                expectedCacheDuration,
+                reload,
+                questionRuntime,
+              }).as("cacheDuration");
+            });
+
+            cy.then(function () {
+              expect(this.freshResult.length).to.eq(32);
+              expect(this.cachedResult.length).to.eq(32);
+              expect(this.freshResult).to.not.eq(this.cachedResult);
+              expect(
+                this.cacheDuration,
+                `Cache should last ${humanizeDuration(expectedCacheDuration)}`,
+              ).to.be.approximately(expectedCacheDuration, 2500);
+            });
+          });
+        },
+      );
+    });
+
+    /** Test that when a question is in a dashboard and both the dashboard and the question have a strategy,
+     * the question is cached according to the dashboard's strategy, not the question's strategy
+     *
+     * Note: The shortest possible adaptive strategy invalidates the cache after just a few seconds,
+     * but the shortest possible duration strategy invalidates the cache after an hour,
+     * so it's easier to tell when an adaptive strategy has overridden a duration strategy -
+     * you can just check that cache is invalidated after a few seconds.
+     */
+    it("dashboard's adaptive strategy can override question's duration strategy", () => {
```

Comment:
The expected behavior is actually the opposite, so there is a problem with this test.